### PR TITLE
Testing TicTacToe game

### PR DESCRIPTION
**Pull request by Blaise-Shyaka**
In this project, we have done the following:
- Identified the functionalities of each method in the classes TicTacToe and Player
- Wrote test cases to check if those methods meet functionality expectations

### CODE REVIEW

File: spec/game_logic_spec.rb [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      expect(game.instructions.class).to eq(Array)
+    end
+
+    it 'Should return an array of three elements' do
+      expect(game.instructions.length).to eq(3)
+    end
+
+    it 'Should return an array of strings' do
+      expect(game.instructions.all?(String)).to be true
+    end
```
Comment (kobiyoyo) [inline spec/game_logic_spec.rb]: - [x] test for when no input is provided

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team    
#### Great Job so far, everything works fine, but there are a few more things required. 
### General 
- [x] Lets test for when there is  a tie 
- [x] Check the comment below 
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Comment by atenaiis** [comment]
thank you for the review, 

We worked on the changes requested.

cheers.

**Review by elmejdki — APPROVED** [review]
# status: approved :heavy_check_mark: 
## Great job your tests are covering all the methods congrats.

now you can merge your project is approved.

## Happy Coding :tophat:

**Pull request merged**